packages/navigation/src/Navigation.tsx

```diff
@@ -47,7 +47,7 @@ const Navigation = <T extends object>(props: NavigationProps<T>) => {
     });
   }, [wrapperRef, itemListRef, setCollapse]);
 
```

Comment:
This prevents the flicker that would otherwise occur when the `Navigation` renders as tabs before collapsing down to the `Select`.